### PR TITLE
PLAT-232 Added support for a log path bind mount

### PR DESCRIPTION
**Pull request by CastelloG**
To access logs after instant goes down

### CODE REVIEW

File: cli/docker.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func sliceContains(slice []string, element string) bool {
 	return false
 }
 
-func extractCommands(startupCommands []string) (environmentVariables []string, deployCommand string, otherFlags []string, packages []string, customPackagePaths []string, instantVersion string, targetLauncher string) {
+func extractCommands(startupCommands []string) (environmentVariables []string, deployCommand string, otherFlags []string, packages []string, customPackagePaths []string, instantVersion string, targetLauncher string, logPath string) {
```
[the remark this answers is not in the document]
Comment (CastelloG) [reply cli/docker.go]: Fixed

File: cli/docker.go [hunk 2]
```diff
@@ -99,57 +110,60 @@ func sliceContains(slice []string, element string) bool {
 	return false
 }
 
-func extractCommands(startupCommands []string) (environmentVariables []string, deployCommand string, otherFlags []string, packages []string, customPackagePaths []string, instantVersion string, targetLauncher string) {
-	instantVersion = "latest"
+func extractCommands(startupCommands []string) commandsOptions {
+	var commandOptions commandsOptions
+	commandOptions.instantVersion = "latest"
```
Comment (Mark-Labuschagne) [inline cli/docker.go:115]:
```suggestion
	commandOptions := commandsOptions{
		instantVersion: "latest",
	}
```

**Review by Mark-Labuschagne — COMMENTED** [review]
One minor comment, looks good otherwise

**Review by tumbledwyer — APPROVED** [review]
LGTM

**Pull request merged**